services/gateway/shared/utils.js

```diff
@@ -68,6 +68,8 @@ const transformParams = (type, params) => {
 			};
 		} else if (type === 'PoS') {
 			if (paramKey === 'name') value = { $ref: '#/parameters/validatorName' };
+		} else if (type === 'reward' && paramKey === 'height') {
+			value = { $ref: '#/parameters/blockHeight' };
```

Comment:
Same ab above. This probably can be avoided.

Reply:
https://github.com/LiskHQ/lisk-service/pull/1373#discussion_r1042284866